src/binder_network.c

```diff
@@ -2014,6 +2051,43 @@ binder_network_modem_reset_cb(
     binder_network_reset_initial_attach_apn(self);
 }
 
+static
+void
+binder_network_current_physical_channel_configs_cb(
+    RadioClient* client,
+    RADIO_IND code,
+    const GBinderReader* args,
+    gpointer user_data)
+{
+    BinderNetworkObject* self = THIS(user_data);
+    GBinderReader reader;
+    gboolean nr_connected = FALSE;
+
+    gbinder_reader_copy(&reader, args);
+
+    switch (code) {
+    case RADIO_IND_CURRENT_PHYSICAL_CHANNEL_CONFIGS_1_4: {
```

Comment:
Since this callback is registered to receive only `RADIO_IND_CURRENT_PHYSICAL_CHANNEL_CONFIGS_1_4` events, there's no need to handle anything else.

At most, there may be `if (code == RADIO_IND_CURRENT_PHYSICAL_CHANNEL_CONFIGS_1_4)`, IMO that would be significantly more readable than a `switch` with a block after a `case` followed by `default:`.

Reply:
Simplified the code to use if instead of switch-case.